sdk/python/setup.py

```diff
@@ -37,7 +37,8 @@
     'click',
     'Deprecated',
     'strip-hints',
-    'docstring-parser>=0.7.3'
+    'docstring-parser>=0.7.3',
+    'colorama'
```

Comment:
I think it would be good to look the version of `colorama` in order to avoid future problems with versions.

Reply:
Added version constraint to the `colorama>=0.4.4` dependency.

Reply:
Maybe it we don't need to have it as a hard requirement since: 

``` Note for colorama support: Click will automatically detect when colorama is available and use it. Do not call colorama.init()!``` from https://click.palletsprojects.com/en/5.x/utils/ and could keep it only in the requirement.txt

Reply:
I was thinking the other way around, add the dependency to `setup.py` and remove it in `requirements.txt` if not required.

If `colorama` is not installed in the `kfp` cli's python environment, we wouldn't able to highlight the cli command output text.